ebt/Dialog/BtReciever.vala

```diff
@@ -203,36 +209,37 @@ public class BtReciever : Granite.Dialog {
             return;
         }
         if (elapsed_time == 0) {
-		    return;
-		}
-		uint64 transfer_rate = transferred / elapsed_time;
-		if (transfer_rate == 0) {
-		    return;
-		}
-		rate_label.label = _("<b>Rate:</b> %s").printf (GLib.format_size (transfer_rate));
-		uint64 remaining_time = (total_size - transferred) / transfer_rate;
-		progress_label.label = _("Receiving…  %s of %s remaining %s").printf (GLib.format_size (transferred), GLib.format_size (total_size), format_time((int)remaining_time));
+            return;
+        }
+        uint64 transfer_rate = transferred / elapsed_time;
+        if (transfer_rate == 0) {
+            return;
+        }
+        rate_label.label = _("<b>Transfer rate:</b> %s").printf (GLib.format_size (transfer_rate));
+        uint64 remaining_time = (total_size - transferred) / transfer_rate;
+        progress_label.label = _("Receiving… %s of Size: %s, remaining time %s").printf (GLib.format_size (transferred), GLib.format_size (total_size), format_time ((int)remaining_time));
```

Comment:
It would be more accurate to say `_("%s of %s received, time remaining %s")`  I think you do not need to say "Size:" in English but if you think translators need help, then add a comment explaining what each placeholder represents. E.g.
`///TRANSLATORS The first placeholder represents the number of bytes received, the second the total number of bytes to be transferred and the third the estimated time remaining before the transfer completes`

Reply:
Ok I will fix soon